usermods/INA219_v2/Readme.md

```
@@ -0,0 +1,130 @@
+
+		| Address             | Description                        | Value   |
+		|---------------------|------------------------------------|---------|
+		| `0x40`              | 0x40 - Default                     | 0x40    |
+		| `0x41`              | 0x41 - A0 soldered                 | 0x41    |
+		| `0x44`              | 0x44 - A1 soldered                 | 0x44    |
+		| `0x45`              | 0x45 - A0 and A1 soldered          | 0x45    |
+
+# Compiling
+
+To enable, add `INA219_v2` to your `custom_usermods`  (e.g. in `platformio_override.ini`)
+```ini
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)